iTechArt.Hotels.Database/Accounts.sql

```diff
@@ -1,6 +1,7 @@
-﻿CREATE TABLE [dbo].[Accounts]
-(
-    [Id] INT NOT NULL PRIMARY KEY IDENTITY, 
-    [Email] VARCHAR(40) NOT NULL, 
-    [Password] VARCHAR(20) NOT NULL
-)
+﻿CREATE TABLE [dbo].[Accounts] (
+    [Id]       INT          IDENTITY (1, 1) NOT NULL,
+    [Email]    VARCHAR (125) NOT NULL,
```

Comment:
Why do you think 125 will be enough?

Reply:
Biggest available email is 320 symbols, recommended is between 50 and 125; Could make email size bigger, but I really think 125 is enough